I got build error:
```
C:/msys32/home/pabbott/lumitec-dev-fw-plibox-esp32/esp-eth-libesphttpd/main/cgi_common.c: In function 'cgiJsonResponseCommonMulti':
C:/msys32/home/pabbott/lumitec-dev-fw-plibox-esp32/esp-eth-libesphttpd/main/cgi_common.c:95:26: error: 'HTTPD_MAX_SENDBUFF_LEN' undeclared (first use in this function)
   size_t max_send_size = HTTPD_MAX_SENDBUFF_LEN / 2; // div/2 to keep some room for headers
                          ^
C:/msys32/home/pabbott/lumitec-dev-fw-plibox-esp32/esp-eth-libesphttpd/main/cgi_common.c:95:26: note: each undeclared identifier is reported only once for each function it appears in
make[1]: *** [/home/pabbott/lumitec-dev-fw-plibox-esp32/Tools/esp-idf/make/component_wrapper.mk:290: cgi_common.o] Error 1
```
Looks like I was using that macro in my own cgi function.  Can we keep that macro available to avoid breaking peoples code?  
I think anyone using that macro probably wants the -2 bytes value.
I propose:
1. Add additional define for convenience: `#define HTTPD_MAX_SENDBUFF_LEN     HTTPD_SENDBUFF_MAX_FILL`
2. Or change name globally of HTTPD_SENDBUFF_MAX_FILL to HTTPD_MAX_SENDBUFF_LEN

I'll do # 1 and continue testing...

--- a/include/libesphttpd/httpd.h
+++ b/include/libesphttpd/httpd.h
@@ -39,7 +39,12 @@ extern "C" {
 
 //Send buffer limit for httpdSend. 2 bytes are reserved for chunk termination ('\r\n').
 #ifndef HTTPD_SENDBUFF_MAX_FILL
-#define HTTPD_SENDBUFF_MAX_FILL	HTTPD_SENDBUFF_SIZE - 2
+#define HTTPD_SENDBUFF_MAX_FILL	(HTTPD_SENDBUFF_SIZE - 2)
+#endif
+
+//Send buffer limit (for backward compatibility
+#ifndef HTTPD_MAX_SENDBUFF_LEN
+#define HTTPD_MAX_SENDBUFF_LEN HTTPD_SENDBUFF_MAX_FILL
 #endif
 
 //If some data can't be sent because the underlaying socket doesn't accept the data (like the nonos

